### DISCUSSION TITLE
vscode-opc-cad-viewer on Mac mini M1 well installed, but drawings are not shown

### DISCUSSION THREAD

**Discussion by hehe-coder** [bernhard-42/vscode-ocp-cad-viewer#63] [Q&A]
Thank you for providing that great work Bernhard. 

I have followed the installation advices for the apple silicon devices. Everything looks well. The Python scripts for CadQuery and  also for build123d run without an error. The Oct-cad-viewer ist running and is present on the screen. But the Viewer-Window stayes empty. Any idea?
<img width="1216" alt="Screenshot" src="https://github.com/bernhard-42/vscode-ocp-cad-viewer/assets/60174538/62f4aa1c-aed1-401d-b05c-4e85bdd75877">

**Comment by bernhard-42** [comment]
I guess, you already tried restarting.
Did you open a file or a folder? Folder provide a proper root workspace path and the viewer is more reliable.

Do you see any errors in "Ausgabe" under "OCP CAD Viewer"?
If not, try to use the command menu of VS Code to,"toggle developer tools". This will open the javascript console. Maybe we find something there.

**Comment by hehe-coder** [comment]
Hallo, thank you for the quick answer. The hint to look at the developer tools was the key to find the reason that the Viewer does not work in my environment. First I have to apologize. I did not mention that I installed in a virtual Parallels Mac environment. This environment doesn't support WebGL in a proper way. I finally run the WebGL Conformance Test (https://registry.khronos.org/webgl/sdk/tests/webgl-conformance-tests.html). The result explicitly showed that WebGL is not supported.
I now will install VS Code directly on my Mac.  Many thanks for your support!

**Comment by bernhard-42** [comment]
glad you found the root cause and hope you'll enjoy the viewer
